ui/modal/modalPublishPreview/view.jsx

```diff
@@ -151,37 +152,43 @@ const ModalPublishPreview = (props: Props) => {
   const txFee = previewResponse ? previewResponse['total_fee'] : null;
   //   $FlowFixMe add outputs[0] etc to PublishResponse type
   const isOptimizeAvail = filePath && filePath !== '' && isVid && ffmpegStatus.available;
-  let modalTitle;
-  if (isStillEditing) {
-    if (livestream) {
-      modalTitle = __('Confirm Update');
-    } else {
-      modalTitle = __('Confirm Edit');
-    }
-  } else if (livestream) {
-    modalTitle = releasesInFuture ? __('Schedule Livestream') : __('Create Livestream');
-  } else {
-    modalTitle = __('Confirm Upload');
-  }
 
-  let confirmBtnText;
-  if (!publishing) {
+  const [modalTitle, setModalTitle] = React.useState('Upload');
+  const [confirmBtnText, setConfirmBtnText] = React.useState('Save');
+
+  React.useEffect(() => {
     if (isStillEditing) {
-      confirmBtnText = __('Save');
-    } else if (livestream) {
-      confirmBtnText = __('Create');
+      if (livestream || isLivestreamClaim) {
+        setModalTitle(__('Confirm Update'));
+      } else {
+        setModalTitle(__('Confirm Edit'));
+      }
+    } else if (livestream || isLivestreamClaim) {
+      setModalTitle(releasesInFuture ? __('Schedule Livestream') : __('Create Livestream'));
+    } else if (isMarkdownPost) {
+      setModalTitle(__('Confirm Post'));
     } else {
-      confirmBtnText = __('Upload');
+      setModalTitle(__('Confirm Upload'));
```

Comment:
- [ ] When publishing a replay, it says "Create Livestream" ... was expecting "Confirm Upload" like in production since it's no longer live.  But this is minor.
- [ ] The other part is that the "View my dashboard" modal appears.  Production goes straight to Uploads Page.

Reply:
Your 2nd point should already be fixed. Are you on the latest version?